tests/test_coverage_gaps.py

```
@@ -0,0 +1,1276 @@
+class TestDisconnectBleInterfaceCloseSyncAwaitable:
+    """Cover ble.py lines 1016-1017: _close_sync awaitable path."""
+
+    def test_close_returns_awaitable(self):
+        from mmrelay.meshtastic.ble import _disconnect_ble_interface
+
+        iface = MagicMock()
+        iface._exit_handler = None
+        iface.disconnect.return_value = None
+        iface.client = None
+
+        async def async_close():
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `async_close`

Add return type annotation: `None`

(ANN202)